src/darwin/Framework/CHIP/MTRDevice_Concrete.mm

```diff
@@ -514,6 +514,12 @@ - (nullable NSNumber *)productID
     return [_pid copy];
 }
 
+- (MTRNetworkCommissioningFeature)networkCommissioningFeatures
+{
+    std::lock_guard lock(_descriptionLock);
+    return [_allNetworkFeatures unsignedIntValue];
```

Comment:
So this will return a value with no bits set when the actual state might be "we don't know yet".  Is that expected?

That is, should we differentiate in the API between "we don't know yet" and "we know and it supports nothing"?